src/fields/ColorField/index.ts

```diff
@@ -1,45 +0,0 @@
-import type { Field, TextFieldValidation } from 'payload'
-
-export const colorField: (name: string) => Field = (name) => {
-  return {
-    name: name,
-    type: 'text',
-    required: true,
-    // TODO: admin config for custom view that shows off the color, picker or tailwind panels for choosing
-    validate: validateHSLValue,
-  }
-}
-
-const validateHSLValue: TextFieldValidation = (value: string | null | undefined): string | true => {
```

Comment:
Seeing this reminded me we still need to do some work on the "new" color picker. This is probably not a part of this PR but I made an issue (#282) and mentioned this PR to be inspired by this code.